mesheryctl/helpers/component_info.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "mesheryctl",
   "type": "client",
-  "next_error_code": 1128
```

Comment:
do no manually update and define error codes.

Reply:
> Do not manually update and define error codes.
> 
> No need to define these yourself, as your PR gets merged, workflow will automatically take care of replacing the Error code in go file as well as updating the component_info.json and bunch of other files.

this was my bad, did not know about this automation step